templates/Template.CSharp/Contract1.cs

```diff
@@ -5,6 +5,9 @@
 
 namespace $safeprojectname$
 {
+    [ManifestExtraAttribute("Author", "Neo")]
```

Comment:
We can use the short way

```suggestion
    [ManifestExtra("Author", "Neo")]
```